$NetBSD: patch-Makefile,v 1.1 2020/08/31 20:01:12 js Exp $

-static does not work on macOS, and there is no reason why this needs
to be static.

--- Makefile.orig	2020-02-14 21:38:03.000000000 +0000
+++ Makefile
@@ -1,6 +1,5 @@
 VERSION=1.10.1
 CFLAGS+=-g -DVERSION='"$(VERSION)"' -Wall -Wextra -Werror -Wno-unused-parameter
-LDFLAGS+=-static
 INCLUDE+=-Iinclude
 PREFIX?=/usr/local
 _INSTDIR=$(DESTDIR)$(PREFIX)
